build.mill

```diff
@@ -388,7 +389,12 @@ trait MillJavaModule extends JavaModule {
 
   // Workaround for Zinc/JNA bug
   // https://github.com/sbt/sbt/blame/6718803ee6023ab041b045a6988fafcfae9d15b5/main/src/main/scala/sbt/Main.scala#L130
-  def testArgs: T[Seq[String]] = Task { Seq("-Djna.nosys=true") }
+  def testArgs: T[Seq[String]] = Task {
+    val userLang =
+      if (Properties.isMac || Properties.isWin) Seq("-Duser.language=en")
+      else Nil
+    Seq("-Djna.nosys=true") ++ userLang
+  }
```

Comment:
Can you keep the comment (above this block) close to the `-Djna.nosys=true`, where it belongs.